sass/includes/_record-matters.scss

```diff
@@ -63,13 +63,14 @@
     &__buttons {
         margin-top: 1rem;
         text-align: center;
-        display: grid;
+        display: flex;
         grid-template-columns: 1fr 1fr 1fr;
         grid-gap: 1.5rem;
```

Comment:
If we change to `display: flex;` then we don't need `grid-template-columns` and `grid-gap`

Reply:
I think we need to keep `grid-gap: 1.5rem;` in, as the buttons will come together without it. Unless there's a way that we can add standardised margins to buttons?

Reply:
Does `grid-gap` do anything without `display: grid;`?

Reply:
`grid-gap` will still apply with `display:flex`
https://css-tricks.com/minding-the-gap/

Reply:
Wow - I didn't know that. Thanks for the link 🙌🏼